src/client/utils/validations.js

```diff
@@ -245,19 +281,19 @@ function isValidDeductibleExpenses(data) {
     isValidField(isValidMonetaryValue, data.deductibleEducationExpenses);
 }
 
-function isValidAdditionalInformation(data) {
+function isValidVAFacility(data) {
   return validateIfDirty(data.facilityState, isNotBlank) &&
     validateIfDirty(data.vaMedicalFacility, isNotBlank);
 }
 
 function isValidMedicareMedicaid(data) {
-  return isBlankDateField(data.medicarePartAEffectiveDate) ||
-    isValidDateField(data.medicarePartAEffectiveDate);
+  return validateIfDirty(data.isMedicaidEligible, isNotBlank) &&
+    validateIfDirty(data.isEnrolledMedicarePartA, isNotBlank) &&
```

Comment:
I think they are independent of each other. The original form has both as required fields with no conditional logic, so I just followed that.